test/residency/recovery/test_journal_public_seam.py

```
@@ -0,0 +1,440 @@
+        raise AssertionError("POSIX journal seam omitted the mbedcrypto fallback")
+
+    environment: Mapping[str, str] = {
+        "NLOHMANN_JSON_INCLUDE_DIR": "/fixture/nlohmann",
+        "MBEDTLS_INCLUDE_DIR": "/fixture/mbedtls",
+        "MBEDCRYPTO_LIBRARY": "/fixture/libmbedcrypto.a",
+    }
+    configured = compiler_command(output, configured=["c++"], environment=environment)
+    for include in ("/fixture/nlohmann", "/fixture/mbedtls"):
+        include_index = configured.index(include)
+        if configured[include_index - 1] != "-I":
+            raise AssertionError(f"POSIX journal seam miswired include {include}")
```

Ruff (0.16.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)